hera_pspec/uvpspec.py

```diff
@@ -301,6 +302,23 @@ def get_integrations(self, key, omit_flags=False):
 
         return self.integration_array[spw][blpairts, polpair]
 
+    def get_r_params(self):
+        """
+        decompress r_params dictionary (so it can be readily used).
```

Comment:
Docstring could be a little more helpful here -- what are r_params, how is it stored etc.

Reply:
I've lengthened this docstring to make it more helpful.